VCFv4.3.tex

```diff
@@ -503,41 +504,87 @@ \subsection{VCF tag naming conventions}
 
 \section{INFO keys used for structural variants}
 \label{sv-info-keys}
-The following INFO keys are reserved for encoding structural variants. In general, when these keys are used by imprecise variants, the values should be best estimates. When a key reflects a property of a single alt allele (e.g. SVLEN), then when there are multiple alt alleles there will be multiple values for the key corresponding to each alelle (e.g. SVLEN=-100,-110 for a deletion with two distinct alt alleles).
+The following INFO keys are reserved for encoding structural variants. When a key reflects a property of a single alt allele (e.g. SVLEN), then when there are multiple alt alleles there will be multiple values for the key corresponding to each alelle (e.g. SVLEN=-100,-110 for a deletion with two distinct alt alleles).
 \footnotesize
 \begin{verbatim}
 ##INFO=<ID=IMPRECISE,Number=0,Type=Flag,Description="Imprecise structural variation">
 ##INFO=<ID=NOVEL,Number=0,Type=Flag,Description="Indicates a novel structural variation">
 ##INFO=<ID=END,Number=1,Type=Integer,Description="End position of the variant described in this record">
 \end{verbatim}
 \normalsize
-For precise variants, END is POS + length of REF allele - 1, and the for imprecise variants the corresponding best estimate.
+For precise variants, END is POS + length of REF allele - 1.
```

Comment:
@SHuang-Broad nicely clarifies the distinction between (A) not having enough sequence information in the vicinity of the breakpoint and (B) not being able to identify the precise breakpoint (despite having complete sequence information) due to a stretch of homology. (A) is an information problem (ie not enough data), while (B) is a methodology problem that cannot be solved no matter how many times you sequence across the breakpoint (because it is already information-complete). For this reason (B) can be addressed only by (1) acknowledging the intractability of identifying a single coordinate in all variants with sequence homologies, or (2) enforcing a convention such as the left-justification one @Shuang-Broad proposes.